src/libsync/discovery.cpp

```diff
@@ -539,18 +539,18 @@ void ProcessDirectoryJob::processFileAnalyzeRemoteInfo(
             } else {
                 item->_instruction = CSYNC_INSTRUCTION_SYNC;
             }
-        } else if (dbEntry._modtime <= 0 && serverEntry.modtime > 0) {
+        } else if ((dbEntry._modtime <= 0 && serverEntry.modtime > 0) || dbEntry._modtime != serverEntry.modtime) {
```

Comment:
@mgallien Doesn't `dbEntry._modtime != serverEntry.modtime` also cover the case when `(dbEntry._modtime <= 0 && serverEntry.modtime > 0)`? If this is the case, can we just go with `if(dbEntry._modtime != serverEntry.modtime)`?

Reply:
yes